src/requestHandler.ts

```diff
@@ -59,7 +59,7 @@ type GetLDUser = ({
 
 export const isNotApplicationRoute = (path) => {
   const isInNextPath = path.match(/^\/_next\/(?!data)/)
-  const isFileNotInNextDataPath = path.match(/^\/_next\/(?!data)/)
+  const isFileNotInNextDataPath = path.match(/(?<!\/_next\/data\/.*)\.\w{1,4}$/)
```

Comment:
can any assets be in the `data` directory? we seem to assume here that assets are always scoped exactly by one level

Reply:
1. No I don't think so https://github.com/vercel/next.js/blob/ad22e7730919f37c5d1eac44d2e73a47f94d1a5e/packages/next/next-server/server/next-server.ts#L407
2. What do you mean by scoping assets by one level?

Reply:
I messed up reading the regex and interpreted `\/.*` was a folder as it would be in... `\/.*\/`. yours matches any amount of folders, so we're goooood